components/raftstore/src/store/worker/pd.rs

```diff
@@ -778,17 +778,17 @@ where
     #[inline]
     pub fn maybe_send_read_stats(&self, read_stats: ReadStats) {
         if let Some(sender) = &self.read_stats_sender {
-            if sender.send(read_stats).is_err() {
-                debug!("send read_stats failed, are we shutting down?")
+            if sender.try_send(read_stats).is_err() {
```

Comment:
Maybe also write the error into the log

Reply:
inapposite. 
If the channel is full, and the logging will be unstoppable.